composer.json

```diff
@@ -28,6 +28,7 @@
         "bower-asset/jquery-form": "^3.51",
         "bower-asset/jquery-colorbox": "^1.5.14",
         "bower-asset/sprintf": "~1.0.3",
+        "css-crush/css-crush": "^2.4.0",
```

Comment:
See if you can add this without updating all our dependencies. Would rather project-wide updates happen in a separate PR.